conf.py

```diff
@@ -7,22 +7,22 @@
     "sphinx_markdown_tables",
     "sphinx_tabs.tabs",
     "sphinxcontrib.httpdomain",
-    "sphinxext.opengraph"
+    "sphinxext.opengraph",
+    "furo50",
 ]
 
 html_css_files = [
-    "/_static/custom.css?" + str(round(time.time())),
-    "https://use.fontawesome.com/releases/v5.13.0/css/all.css"
+    "custom.css",
```

Comment:
The theme automatically adds the cache versioning so we no longer have to do it manually.
![image](https://github.com/user-attachments/assets/ec8217a6-f777-47bd-b4e1-06aa0bf5df69)
![image](https://github.com/user-attachments/assets/c3518a28-a9fc-44db-8674-54c964c170eb)